capstone/src/main/java/com/google/sps/servlets/clubs/ClubServlet.java

```diff
@@ -124,4 +149,40 @@ private PreparedQuery retrieveClub(HttpServletRequest request, DatastoreService
     PreparedQuery prepared = datastore.prepare(query);
     return prepared;
   }
+
+  /** Return the Calendar ID after creating a calendar for the given club name. */
+  public String createCalendar(String clubName, Calendar service)
+      throws IOException, GeneralSecurityException {
+    com.google.api.services.calendar.model.Calendar calendar =
+        new com.google.api.services.calendar.model.Calendar()
+            .setSummary(clubName + " Calendar")
+            .setTimeZone("America/Los_Angeles");
```

Comment:
I'm dealing with time zones in my code right now - maybe we can figure out a way to get the user's current itme zone, instead of hard-coding it? In the meantime, maybe we can extract time zone to a Constant.

Reply:
That's a good idea, thanks for bringing it up! I made it a constant for now.